src/components/SideNav/index.jsx

```diff
@@ -15,7 +15,8 @@ const SideNav = (props) => {
       <Link to={{ pathname: `${BASE_URL}/resources` }} className="ListItem">Infrastructure</Link>
       <div>
         <NavLink to={{ pathname: `${BASE_URL}/nodes` }} activeClassName="active" className="SubListItem">Nodes</NavLink>
-        <NavLink to={{ pathname: `${BASE_URL}/pvcs` }} className="SubListItem">Volumes</NavLink>
+        <NavLink to={{ pathname: `${BASE_URL}/pvs` }} className="SubListItem">Volumes</NavLink>
```

Comment:
@Mubangizi the link for volumes is defined as `${BASE_URL}/volumes` and not `${BASE_URL}/pvs` as you seen in this PR

Reply:
Ohh, I dint Know what it was since its not on `develop-new` yet.